R/txtq.R

```diff
@@ -113,6 +147,17 @@ R6_txtq <- R6::R6Class(
         quote = FALSE
       )
     },
+    txtq_reset = function(){
+      unlink(private$db_file, force = TRUE)
+      fs::file_create(private$db_file)
+      private$txtq_set_head(0)
+      private$txtq_set_total(0)
+    },
+    txtq_clean = function(){
```

Comment:
note: this may not preserve message ordering if something is pushed between lines 158 and 159, right?